docs/en/sql-reference/functions/other-functions.md

```diff
@@ -555,6 +555,42 @@ Result:
 └───────────────┘
 ```
 
+## currentSchemas
+
+Returns a single-element array with the name of the current database.
+
+**Syntax**
+
+```sql
+currentSchemas(bool)
+```
+
+Alias: `current_schemas`.
+
+**Arguments**
+
+- `bool`: A boolean value. [Bool](../data-types/boolean.md).
+
+:::note
+The argument is actually ignored, but for compatibility's sake with the implementation of this function in other DB engines it is required.
```

Comment:
`The boolean argument is ignored. It only exists for the sake of compatibility with the implementation of this function in PostgreSQL.`

\+ let's link `implementation` to https://www.postgresql.org/docs/7.3/functions-misc.html